pkg/runtime/master.go

```diff
@@ -41,6 +41,11 @@ func (k *KubeadmRuntime) InitMaster0() error {
 	if err != nil {
 		return fmt.Errorf("add apiserver domain hosts failed %v", err)
 	}
+	// fix issue https://github.com/labring/sealos/issues/1621
+	err = k.sshCmdAsync(k.getMaster0IPAndPort(), "sed -i \"/update_etc_hosts/c \\ - ['update_etc_hosts', 'once-per-instance']\" /etc/cloud/cloud.cfg && touch /var/lib/cloud/instance/sem/config_update_etc_hosts")
```

Comment:
I don't think this is appropriate. We should give up a solution instead of implementing it directly in the code, such as the "create_vm_sealos.sh` or `sealos-vm" we mentioned to solve it.

Reply:
https://github.com/labring/sealos-vm/blob/627a2ac6904e4c83eb9f3175e51d5e3bdd23c610/pkg/apply/infra/mulitipass/init.go#L257  This logic has been implemented.

Reply:
Seems only multipaas has this config? `/etc/cloud/cloud.cfg`

Reply:
> Seems only multipaas has this config? `/etc/cloud/cloud.cfg`

Cloud-init has this configuration, such as multipaas or lima.